modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -647,58 +648,49 @@ definition {
 				actionLabel = "Notification action when entry is added",
 				active = "true",
 				inputExpression = '''isEmpty(oldValue("customField"))''',
-				notificationName = "Email Template",
+				notificationName = "Notification Template",
 				objectLabel = "Custom Object 175197",
 				thenAction = "Notification",
 				whenAction = "On After Add");
 		}
 
-		task ("When: An entry is added") {
+		task ("When an entry is added") {
 			ObjectAdmin.goToCustomObject(objectName = "CustomObject175197");
 
 			ObjectPortlet.addSingleFieldEntryViaUI(entry = "Entry Test");
 		}
 
-		task ("Then: The action is triggered and the notification is sent") {
-			Notifications.notificationQueueAdmin();
-
-			AssertElementPresent(
-				key_objectName = "CustomObject175197",
-				locator1 = "ObjectNotifications#OBJECT_QUEUE_NAME");
-
-			ObjectAdmin.openObjectAdmin();
-
-			ObjectPortlet.selectCustomObject(label = "Custom Object 175197");
-
-			ObjectAdmin.goToActionsTab();
-
-			ObjectAdmin.assertObjectActions(
-				actionActive = "Yes",
-				actionLabel = "Notification action when entry is added",
-				status = "Success");
+		task ("Then the action is triggered and the notification is sent") {
+			MockMock.viewMail(
+				asset = "Subject Test",
+				content = "Email Body",
+				recipient = "test@liferay",
+				sender = "Test Test",
+				subject = "Subject Test");
```

Comment:
You're combining two different changes here:
- Updating the wording of the strings
- Changing the logic of the tests to use `MockMock.viewMail`

These should be two separate commits because they're completely separate changes. Please do the following:
- Update all strings in one commit using the commit message `LPS-181500 Wordsmith`
- Update the tests to use `MockMock.viewMail` in a different commit